<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nTrainClass</class>
 <widget class="QDialog" name="nTrainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>neuroTrainer</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>329</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Train directory</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbConvert">
          <property name="text">
           <string>Convert train files</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Epochs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbEpochs">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pbTest">
          <property name="text">
           <string>Test</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Threads</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbThreads">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Save path</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pbSave">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="leTrainPath">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbBrowseDir">
          <property name="text">
           <string>Browse Train Folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbTrain">
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="text">
           <string>Train</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbStop">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leOpenPath">
          <property name="text">
           <string>./bpn1.bpn</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbBrowseFile">
          <property name="text">
           <string>Load BPN</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="teLog">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="pbProgress">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leTrainPath</tabstop>
  <tabstop>pbConvert</tabstop>
  <tabstop>pbBrowseDir</tabstop>
  <tabstop>sbEpochs</tabstop>
  <tabstop>pbTrain</tabstop>
  <tabstop>pbTest</tabstop>
  <tabstop>pbStop</tabstop>
  <tabstop>leOpenPath</tabstop>
  <tabstop>pbSave</tabstop>
  <tabstop>pbBrowseFile</tabstop>
  <tabstop>teLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
